From d968bbe73d04fc76db30bdefc9b00fb4128d6b4e Mon Sep 17 00:00:00 2001
From: Manish Pandey <quic_mapa@quicinc.com>
Date: Fri, 13 Oct 2023 19:38:59 +0530
Subject: [PATCH] QCLINUX: arm64: dts: qcom: qcm6490: disable sdhc1 for ufs
 target

Disable sdhc1 for QCM6490 for ufs boot target to avoid probe
for sdhc1 as vreg_l7b_2p9 is shared regulator for both ufs vcc
and emmc vcc. Currently this is causing probe failure for ufs.

Signed-off-by: Manish Pandey <quic_mapa@quicinc.com>
Signed-off-by: Salendarsingh Gaud <quic_sgaud@quicinc.com>
Signed-off-by: Atul Dhudase <quic_adhudase@quicinc.com>
Upstream-Status: Pending
---
 arch/arm64/boot/dts/qcom/qcm6490-idp.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/qcom/qcm6490-idp.dts b/arch/arm64/boot/dts/qcom/qcm6490-idp.dts
index 424cd9c2b092..7f682093e954 100644
--- a/arch/arm64/boot/dts/qcom/qcm6490-idp.dts
+++ b/arch/arm64/boot/dts/qcom/qcm6490-idp.dts
@@ -450,7 +450,7 @@ &sdhc_1 {
 	vmmc-supply = <&vreg_l7b_2p952>;
 	vqmmc-supply = <&vreg_l19b_1p8>;
 
-	status = "okay";
+	status = "disabled";
 };
 
 &tlmm {
-- 
2.25.1

